core/webengine.py

```diff
@@ -56,6 +58,8 @@ def __init__(self, buffer_id):
         self.cookie_storage = BrowserCookieStorage(self.config_dir)
         self.cookie_store.cookieAdded.connect(self.cookie_storage.add_cookie)
 
```

Comment:
You can define ```self.hovered_url``` before linkHovered signal connect.

Reply:
thanks!